root_node:
  ros__parameters:
    # Use high frequency topics
    hfreq: true

    # Print diagnostics
    info_imu: false
    info_motor: false

    # Number of robot motors
    num_motors: 29

    # Joint names in the description file
    joint_names:
      - left_hip_pitch_joint
      - left_hip_roll_joint
      - left_hip_yaw_joint
      - left_knee_joint
      - left_ankle_pitch_joint
      - left_ankle_roll_joint
      - right_hip_pitch_joint
      - right_hip_roll_joint
      - right_hip_yaw_joint
      - right_knee_joint
      - right_ankle_pitch_joint
      - right_ankle_roll_joint
      - waist_yaw_joint
      - waist_roll_joint
      - waist_pitch_joint
      - left_shoulder_pitch_joint
      - left_shoulder_roll_joint
      - left_shoulder_yaw_joint
      - left_elbow_joint
      - left_wrist_roll_joint
      - left_wrist_pitch_joint
      - left_wrist_yaw_joint
      - right_shoulder_pitch_joint
      - right_shoulder_roll_joint
      - right_shoulder_yaw_joint
      - right_elbow_joint
      - right_wrist_roll_joint
      - right_wrist_pitch_joint
      - right_wrist_yaw_joint
